hue/hue.sh

```diff
@@ -71,7 +71,7 @@ function install_hue_and_configure() {
   install_packages hue || err "Failed to install Hue"
```

Comment:
Any chance we can get this to install a newest version of Hue? (ie: 4.10)

Reply:
No, newer version of Hue will be available with new Dataproc 2.1 image later this year.